### ISSUE TITLE
Query > Submit query : Error 500

### ISSUE THREAD

**Issue by williamdes** [phpmyadmin/phpmyadmin#14298]
### Steps to reproduce 1
1. https://demo.phpmyadmin.net/master-config/db_multi_table_query.php?db=world
2. Submit query
### Steps to reproduce 1bis
1. https://demo.phpmyadmin.net/master-config/db_multi_table_query.php?db=world
2. Select table
3. Submit query

### Expected behaviour
Case 1: Show a warning or do like https://demo.phpmyadmin.net/QA_4_8/db_multi_table_query.php?db=world
Case 1bis: Update and submit the query

### Actual behaviour
Error 500

### Server configuration
**phpMyAdmin version:** 5.0.0

**Comment by nulll-pointer** [comment]
I can't reproduce in 5.0.0-dev, also, you mentioned version 5.0.0 in the server configuration whereas the links you posted are of the demo server using 4.8.0.

**Comment by williamdes** [comment]
@nulll-pointer The 4.8.0 link shows the _"expected"_ behavior.

Videos:
[Video 1](https://drive.google.com/open?id=1kWoH5AQv8PYq9SdmxdTtwxHT1eYlV0d8)
[Video 2](https://drive.google.com/open?id=11rxsrbasOpL_kILszZ_aHh47chvpS8GR)

Can you reproduce using those videos ?

**Comment by nulll-pointer** [comment]
@williamdes, thanks for the video explanation.
In my case, it gives the following error-
![query_build_error](https://user-images.githubusercontent.com/20344314/40319763-e62736ec-5d46-11e8-8411-98aacef4a81c.png)

**Comment by williamdes** [comment]
I know why you have a blank page, see https://stackoverflow.com/a/13934612/5155484
If you use nginx, in `/etc/php/7.2/fpm/pool.d/www.conf`, add :
```
php_flag[display_errors] = on
php_flag[display_startup_errors] = on
```

**Comment by nulll-pointer** [comment]
I'm not using nginx,
Server configuration : Apache/2.4.29 
php version  : 7.1.14
I double checked the php configuration file and I have set display errors to be on everywhere.
Still seeing the blank page.

**Comment by williamdes** [comment]
set them to off and you will have the same behavior as on my server.

**Comment by mauriciofauth** [comment]
Closed by 87bf7c397f4513a17b3bff9dc643f7442367146b
